manifests/r/rcmaehl/MSEdgeRedirect/0.6.3.0/rcmaehl.MSEdgeRedirect.locale.en-US.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: rcmaehl.MSEdgeRedirect
+PackageVersion: 0.6.3.0
+PackageLocale: en-US
+Publisher: rcmaehl
+PublisherUrl: https://msedgeredirect.com
+PublisherSupportUrl: https://github.com/rcmaehl/MSEdgeRedirect/issues
+Author: 'Publisher: rcmaehl'
+PackageName: MSEdgeRedirect
+PackageUrl: https://msedgeredirect.com
+License: Robert Maehl, using LGPL 3 License
+LicenseUrl: https://github.com/rcmaehl/MSEdgeRedirect/blob/main/LICENSE
+ShortDescription: A Tool to Redirect News, Search, Widgets, Weather and More to Your Default Browser
+Moniker: MSEdgeRedirect
+Tags:
+- Redirect
+ReleaseNotes: 0.6.3.0

```

Comment:
```suggestion
Moniker: msedgeredirect
Tags:
- redirect
ReleaseNotes: |
 * Fixed Edge Profile issues
 * Fixed Settings Start Menu Item
 * Fixed old as heck misspelling in Installer
 * Fixed Start Search Suggestions not working
 * Minor Redesign to Mode and Settings Menu
 * Fixed --from-installer tag being redirected
 * Inactive code for de-embedding news articles
 * Fixed Silent/Deployed Installs missing Start Menu entries
 * Added Cleanup Script to GitHub Assets folder for botched installs
 * Fixed HTTP/HTTPS wanting an app to be selected despite no changes 
```

Reply:
Yeah that makes alot of sense, should've known to use | followed by the changelog itself obviously